docs/de-DE/guide/README.md

```diff
@@ -157,7 +157,7 @@
    scoop install starship
    ```
 
-1. Füge das init-Skript zur Konfigurationsdatei deiner Shell hinzu:
+2. Füge das init-Skript zur Konfigurationsdatei deiner Shell hinzu:
```

Comment:
Using `1.` many times in a row is the markdown way of creating an incrementing list, as shown here:

1. 
1.
1.

![image](https://user-images.githubusercontent.com/4658208/72909683-66613980-3d05-11ea-830b-ffe486caa54a.png)

Reply:
i see :) thanks for the hint :+1: